grpc-stubs/aio.pyi

```diff
@@ -0,0 +1,258 @@
+import typing
+from concurrent import futures
+from types import TracebackType
+from . import (
+    _Options,
+    Compression,
+    GenericRpcHandler,
+    ServerCredentials,
+    StatusCode,
+    ChannelCredentials,
+    CallCredentials,
+)
+
+"""Create Client"""
+
+# FIXME
+ClientInterceptor = typing.Any
```

Comment:
I see a FIXME still hanging around here, could you please update this type to not be an `Any`?

The non-async version of this stub has a type called `Interceptor` that uses a `typing.Union`; is this an equivalent of that, or intended to map to something else?

Reply:
I don't have enough experience with interceptors to know what this type should be, so leaving it as Any allows for future improvements by folks who actually use interceptors. With Any here, one doesn't get any type safety that's true, but also no type errors.

I would rather leave the interceptor definitions unfilled, rather than create incorrect definitions. How can I adjust the comment to clarify this stance?

Reply:
I think I take a different default stance on `typing.Any`. If we get it wrong, it risks introducing a breaking change in the typings later. If we leave it `typing.Any`, it guarantees introducing a breaking change in the typings later.

I need more convincing here.

Reply:
Replaced with an uninhabited type, PTAL